tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs

```diff
@@ -68,5 +68,52 @@ public async Task WordPress51(string phpVersion)
                     Assert.Contains("<title>WordPress &rsaquo; Setup Configuration File</title>", data);
                 });
         }
+
+        [Theory]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        public async Task PhpFpmWordPress51(string phpVersion)
+        {
+            // Arrange
+            string hostDir = Path.Combine(_tempRootDir, "wordpress");
+            if (!Directory.Exists(hostDir))
```

Comment:
There are 2 problems with this and earlier test in this file.
- for all the iterations in **this** test, the same folder is used to build the app. We ideally want unique folder for each variation so that earlier variation's build does not affect the next iteration's build
- This test's folder and earlier test's folder in this file are the same, so we would be having similar issues as mentioned in the above point.

Reply:
I will take a look into this, these tests were originally added in regular php-images and I added tests for fpm images in addition.